where-filter.ts

```diff
@@ -260,8 +270,8 @@ function whereFilter(where) {
 				if (matcher.neq !== undefined && value.length <= 0) {
 					return true;
 				}
-				return value.some(function (v, i) {
-					const cond = {};
+				return value.some(function (_v, i) {
+					const cond = {} as any;
 
 					cond[i] = matcher;
```

Comment:
Why can't this just be `const cond = {[i]: matcher}`? Or just inline that into the `whereFilter(cond)` call

Reply:
yeah, that's better, and avoids at least one `as any` ...done